integration-cli/docker_cli_events_test.go

```diff
@@ -245,3 +245,206 @@ func TestEventsFilters(t *testing.T) {
 
 	logDone("events - filters")
 }
+
+func TestEventsFilterContainerID(t *testing.T) {
+	since := time.Now().Unix()
+	defer deleteAllContainers()
+
+	out, _, err := runCommandWithOutput(exec.Command(dockerBinary, "run", "-d", "busybox", "true"))
+	if err != nil {
+		t.Fatal(out, err)
+	}
+	container1 := stripTrailingCharacters(out)
+	out, _, err = runCommandWithOutput(exec.Command(dockerBinary, "run", "-d", "busybox", "true"))
+	if err != nil {
+		t.Fatal(out, err)
+	}
+	container2 := stripTrailingCharacters(out)
+
+	eventsCmd := exec.Command(dockerBinary, "events", fmt.Sprintf("--since=%d", since), fmt.Sprintf("--until=%d", time.Now().Unix()), "--filter", fmt.Sprintf("container=%s", container1))
```

Comment:
@brahmaroutu can you please do a for loop to reuse code instead of duplicating it?

```
for _, s := range []string { container1, container2, container1[:12], container2[:12] } {
//...
}
```

Thanks!